planner/cascades/testdata/integration_suite_out.json

```diff
@@ -403,6 +403,38 @@
           "4 44",
           "4 44"
         ]
+      },
+      {
+        "SQL": "select t1.a, t1.b from t1 left join t2 on t1.a = t2.a where t1.a > 2 and t2.b > 200",
+        "Plan": [
+          "Projection_16 4166.67 root test.t1.a, test.t1.b",
+          "└─HashLeftJoin_18 4166.67 root left outer join, inner:TableReader_21, equal:[eq(test.t1.a, test.t2.a)]",
+          "  ├─TableReader_19 3333.33 root data:TableScan_20",
+          "  │ └─TableScan_20 3333.33 cop[tikv] table:t1, range:(2,+inf], keep order:false, stats:pseudo",
+          "  └─TableReader_21 3333.33 root data:TableScan_22",
+          "    └─TableScan_22 3333.33 cop[tikv] table:t2, range:(2,+inf], keep order:false, stats:pseudo"
+        ],
+        "Result": [
+          "3 33",
+          "4 44"
```

Comment:
The result is not correct, `4 44` should not in the result.

Reply:
Why? It's a left join.

Reply:
`where t2.b>200` would filter out the unmatched rows.

Reply:
Oh, yes.